docs/source/getting-started/first-feature.rst

```diff
@@ -4,17 +4,15 @@
 Your First Feature
 ******************
 
-Let's dive into implementing a new feature for our social network app.
+Let's dive into implementing a new feature for our social network app. 
```

Comment:
It might be good to configure your editor to remove trailing spaces. Are you using VSCode?
```suggestion
Let's dive into implementing a new feature for our social network app.
```